src/Mod/PartDesign/App/FeatureHelix.h

```diff
@@ -90,6 +90,8 @@ class PartDesignExport Helix : public ProfileBased
 
     // handle changed property
     virtual void handleChangedPropertyType(Base::XMLReader& reader, const char* TypeName, App::Property* prop);
+    static const App::PropertyFloatConstraint::Constraints Helix::floatTurns;
```

Comment:
The "Helix" in this context is redundant (and you don't use it on the next line):
```suggestion
    static const App::PropertyFloatConstraint::Constraints floatTurns;
```

Reply:
do you want to apply the change, @donovaly ? apparently it's just a push of a button now :)